packages/smooth_app/lib/pages/product/summary_card.dart

```diff
@@ -128,58 +131,72 @@ class _SummaryCardState extends State<SummaryCard> {
     // Each Score card takes about 1.5 rows to render.
     totalPrintableRows -= (1.5 * scoreAttributes.length).ceil();
 
-    final List<AttributeGroup> attributeGroupsToBeRendered =
-        _getAttributeGroupsToBeRendered();
+    final List<Widget> displayedGroups = <Widget>[];
+
+    // First, a virtual group with mandatory attributes of all groups
+    final List<Widget> attributeChips =
+        _buildAttributeChips(_getMandatoryAttributes(), false);
```

Comment:
lets use named parameter so the reader knows what `false` is for

Reply:
If you insist I will. But:
1. it's only a private method, of a class that is not in a library
2. the current name of the parameter is `returnNullIfStatusUnknown` and is actually misleading - a more precise naming would be `dontIncludeAnAttributeWhoseStatusIsUnknown` - are you OK with that?

Reply:
yeah `dontIncludeAnAttributeWhoseStatusIsUnknown` sounds could. It helps a lot to have named parameter for stuff like `false` to know what is the value that's being passed

Reply:
That being said there's a side effect, as the way it's coded a label attribute won't be dealt with the same way if it's in the virtual mandatory group (where the flag is `false`) and in the label attribute group (where the flag is `true`).
To be fine-tuned later in a next issue or PR...